provisioning/plugins/corim-nitro-decoder/swcompattributes.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package main
+
+import (
+	"fmt"
+
+	"github.com/veraison/corim/comid"
+)
+
+type NitroSwCompAttributes struct {
+	MeasurementType  string
+	Version          string
+	SignerID         []byte
+	AlgID            uint64
+	MeasurementValue []byte
+}
+
+func (o *NitroSwCompAttributes) FromMeasurement(m comid.Measurement) error {
+	return fmt.Errorf("Not implemented, not needed?")
```

Comment:
ditto, this is needed, but first you need to define your CoMID profile to carry the PCRs